features/supplier/supplier_applies_for_an_opportunity.feature

```diff
@@ -280,3 +280,47 @@ Scenario: Supplier asks a clarification question
   And I click 'Ask question'
   Then I see a success banner message containing 'Your question has been sent.'
 
+@sign-framework-agreement-call-to-action
```

Comment:
I don't think we generally have individual tags per scenario?  Unless there's a reason to start doing this?